python/lsst/dax/imgserv/templates/uws_job_descriptor.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<uws:job xmlns:uws="http://www.ivoa.net/xml/UWS/v1.0">
+  <uws:jobId>{{ uws_job_id }}</uws:jobId>
+  <uws:runId>test</uws:runId>
+  <uws:ownerId/>
+  <uws:phase>{{ uws_job_phase }}</uws:phase>
+  <uws:startTime/>
```

Comment:
Knowing the start and end times would be super helpful, even for debugging.  We will definitely want this.

Reply:
Agree!